src/pages/PostDetails/PostDetails.jsx

```diff
@@ -0,0 +1,43 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { Loading } from "../../components";
```

Comment:
Are you able to use absolute imports ? 
```suggestion
import { Loading } from "components";
```